lambda-api/src/main/resources/cloudformation/api-apikey.yaml

```diff
@@ -915,6 +915,38 @@ Resources:
               - ApiAuthorization: []
               x-amazon-apigateway-integration:
                 $ref: "#/components/x-amazon-apigateway-integrations/lambdaApi200"
+          /documents/compress:
+            post:
+              operationId: CompressDocuments
+              parameters:
+                - $ref: '#/components/parameters/siteIdParam'
+              requestBody:
+                required: true
+                content:
+                  application/json:
+                    schema:
+                      $ref: '#/components/schemas/CompressDocumentsRequest'
+              description: Compress documents into a ZIP
```

Comment:
DocumentsCompressRequest & DocumentsCompressResponse is more consistant than CompressDocumentsRequest/Response

Reply:
Refactored.